tests/ext/root_span_url_with_post_array_allowed.phpt

```diff
@@ -0,0 +1,25 @@
+--TEST--
+Values of a an array are redacted when the array key is in the DD_TRACE_HTTP_POST_DATA_PARAM_ALLOWED env var
```

Comment:
This doesn't really makes sense, you don't want to redact these, basically if the prefix matches exactly, then we just submit it.